jina/drivers/search.py

```diff
@@ -77,6 +77,7 @@ def _apply_all(
                 []
             )  #: missed hit results, some search may not end with results. especially in shards
             for idx, retrieved_doc in enumerate(docs):
+                # some requests would be with old, some with new (in the same batch)
```

Comment:
what does this sentence mean?

Reply:
It doesn't make any sense anymore. It was a thought we had in the middle of the designing.